### PR TITLE
removed build instructions, added links to build in code repo

### PR DESCRIPTION
**Pull request by StarfallProjects**
This is the first part of issue #361: amending the docs to point to the build instructions in the code repo README.

### CODE REVIEW

File: partials/_install-run.md [hunk 1]
```diff
@@ -99,4 +100,6 @@ brew cask install eventstore
 
 In each case you can run Event Store with the `eventstore` command, and stop it with `Ctrl+c`. To use the default database location you need to use `sudo`, or you can change the location with the `--db` parameter.
 
+To build Event Store from source, refer to the [Event Store README](https://github.com/EventStore/EventStore#mac-os-x).
```
Comment (ChrisChinchilla) [inline partials/_install-run.md]: @StarfallProjects This link refers specifically to the macOS build instructions, might be best to link to the general heading?

```suggestion
To build Event Store from source, refer to the [Event Store README](https://github.com/EventStore/EventStore#building-event-store).
```

Comment (StarfallProjects) [reply partials/_install-run.md]: That's for the Mac section of the installation guide. There are links to the Windows build from the Windows section, and Linux from the Linux section.

**Comment by StarfallProjects** [comment]
The README update PR is here: https://github.com/EventStore/EventStore/pull/1979

File: partials/_install-run.md (hunk 1 above)
Comment (ChrisChinchilla) [reply partials/_install-run.md]: Huh, didn't see in the changes. Ok

**Pull request merged**